### PR TITLE
Centralized Typing

### PR DESCRIPTION
**Pull request by Jamesmjkim**
Moved all typing to types.ts and to have all types in a centralized area

removed old utility function

also exported all types

### CODE REVIEW

File: src/denostore.ts [hunk 1]
```diff
@@ -161,7 +162,7 @@ export default class Denostore {
     this.#router.post(this.#route, async (ctx: Context): Promise<void> => {
       const { response, request } = ctx;
       try {
-        const body = await request.body();
+        const body = request.body();
```
Comment (jakedoublev) [inline src/denostore.ts:165]: Can we await request.body().value instead?

File: src/types.ts [hunk 2]
```diff
@@ -37,11 +54,10 @@ export interface ExecutableSchemaArgs<TContext = any> {
   resolvers?: IResolvers<any, TContext> | Array<IResolvers<any, TContext>>; // resolvers for the type definitions
 }
 
-type Maybe<T> = null | undefined | T;
+export type Maybe<T> = null | undefined | T;
```
Comment (jakedoublev) [inline src/types.ts:57]: What is this type here? Maybe<T>?

File: src/utils.ts [hunk 3]
```diff
@@ -1,33 +1,5 @@
 import { QueryObjType, GraphQLResolveInfo } from './types.ts';
 
-// const queryParser = (queryString: string): string => {
```
Comment (jakedoublev) [inline src/utils.ts:3]: Glad to see this ugly algo process gone in favor of such a better solution with the AST's.

**Review by jakedoublev — CHANGES_REQUESTED** [review]
Please address questions.

File: src/denostore.ts (hunk 1 above)
Comment (Jamesmjkim) [reply src/denostore.ts]: resolved

File: src/types.ts (hunk 2 above)
Comment (Jamesmjkim) [reply src/types.ts:57]: It is a generic type. The type given or its going to be null or undefined.

File: src/utils.ts (hunk 3 above)
Comment (Jamesmjkim) [reply src/utils.ts:3]: I am glad to see that go too. AST approach has a faster time complexity!

**Comment by Jamesmjkim** [comment]
Fixed the requested problems

**Review by jakedoublev — APPROVED** [review]
Great work James. These are excellent changes and great typing best practice demonstration.

**Pull request merged**